apps/pdc-frontend/src/app/[locale]/(rootLayout)/[...not-found]/page.tsx

```diff
@@ -8,6 +8,8 @@ import { fetchData } from '@/util/fetchData';
 import { GetNotFoundPageQuery } from '../../../../../gql/graphql';
 
 const NotFoundPage = async ({ params: { locale } }: { params: { locale: string } }) => {
+  const response = new Response(null, { status: 404 });
+  response.headers.set('X-Not-Found', 'true');
```

Comment:
I have never heard of this `X-Not-Found: true` HTTP header. Why would we need it?